back/engines/commercial/analytics/spec/acceptance/analytics_posts_spec.rb

```diff
@@ -6,20 +6,68 @@
 resource 'Analytics - FactPosts model' do
   explanation 'Queries to summarise posts - ideas & initiatives/proposals.'
 
-  example 'group registrations by month' do
-    # do_request({
-    #              query: {
-    #                fact: 'registration',
-    #                groups: 'dimension_date_registration.month',
-    #                aggregations: {
-    #                  all: 'count'
-    #                }
-    #              }
-    #            })
-    # assert_status 200
-    # expect(response_data).to match_array([
-    #                                        { 'dimension_date_registration.month': '2022-09', count: 2 },
-    #                                        { 'dimension_date_registration.month': '2022-10', count: 1 }
-    #                                      ])
+  before do
+    header 'Content-Type', 'application/json'
+    admin_header_token
+  end
+
+  post 'web_api/v1/analytics' do
+    before_all do
```

Comment:
Why is a `before_all` used instead of `before`? We've had some issues with `before_all` in the past (flaky specs).